lightning/src/blinded_path/message.rs

```diff
@@ -74,16 +91,39 @@ impl Writeable for ForwardTlvs {
 	}
 }
 
-impl Writeable for ReceiveTlvs {
-	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), io::Error> {
-		// TODO: write padding
-		encode_tlv_stream!(writer, {
-			(6, self.path_id, option),
-		});
-		Ok(())
-	}
+/// Represents additional data included by the recipient in a [BlindedPath].
+///
+/// This data is encrypted and will be included when sending through
+/// [BlindedPath]. The recipient can authenticate the message and
```

Comment:
Use ticks around identifiers. Seems github converted into formatting when I suggested the change.

Reply:
Done! [653636b](https://github.com/lightningdevkit/rust-lightning/commit/653636b521ec0d4dba73b7aff83e80d594928a9e)
Thanks for catching it!